README.md

```diff
@@ -703,10 +703,28 @@ Default: `true`
 
 #### omitSkippedTests ####
 
-Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCuucumberStyleTestReports is not set to `true`
+Controls whether skipped tests should be completely omitted from test reports rather than showing up as a skip / failure. This mimics the default behavior of Cucumber and will have no effect if outputCucumberStyleTestReports is not set to `true`
 
 Default: `true`
 
+#### pylintDisabledChecker ####
+
+Controls the disabled checkers for Pylint. 
+
+Default: `C,R,W`
+
+#### pylintEnabledChecker ####
+
+Controls the enabled checkers for pylint. 
+
+Default: `E,F`
+
+#### pylintExitZero ####
```

Comment:
S: why not call this something easier to understand like `failOnLintErrors`?

Reply:
Changed, thanks Ryan for the suggestion!